packages/shared/src/components/widgets/SimilarPosts.tsx

```diff
@@ -155,7 +169,7 @@ export default function SimilarPosts({
       <h4 className="py-3 pl-6 pr-4 text-theme-label-tertiary typo-body">
         {title}
       </h4>
-      {Separator}
+      {HorizontalSeparator}
```

Comment:
Why don't you just render as component here?
```
<HorizontalSeparator />
```

Reply:
Not sure, I just updated the name here.

Reply:
Seems like it is a ReactNode, the variable name was just on uppercase. Meaning, we won't be able to render it as `<Separator />`.

Reply:
It is rendered in two places in this same file.

Reply:
Also, the variable is a ReactNode, meaning we can't render is as `<Separator />`.

Reply:
But it's just a div right?
Not sure why we can use it as react component?

Kind of want to move forward with those in favor of variables.

Reply:
For it to become a React component, it needs to be a Function component or class component. There will be no call signature for the variable.